src/Services/FhirOrganizationService.php

```diff
@@ -39,16 +39,27 @@ public function getAll()
                             country_code as country,
                             email
                         FROM facility;";
+
         $facilityResults = sqlStatement($facilitySQL);
+        $results = array();
         while ($row = sqlFetchArray($facilityResults)) {
-            $row['id'] = 'facility-' . $row['id'];
             $row['code'] = 'prov';
             $row['display'] = "Healthcare Provider";
             array_push($results, $row);
         }
         return $results;
     }
-    
+
+    public function getOne($oid)
+    {
+        $facilitySQL = "SELECT id, name, phone, street, city, state, postal_code, country_code as country, email FROM facility WHERE id = ?";
+
+        $result = sqlQuery($facilitySQL, $oid);
```

Comment:
we want to stay consistent and use standard service when possible. Code duplication is not needed in this resource. Take a look at how Yash did the patient service and others when he broke them out of FHIRResourceService.
I explained why in his PR...

Reply:
@sjpadgett Sir, so should I use the existing FacilityService class? In that case, the getAll and getOne queries would return all the columns (~35) of the facility table; but we need only the id, name, contact details and address values.

Reply:
really, that many:) No, you can leave it as you have.